client/extensions/happychat/state/reducers/README.md

```diff
@@ -0,0 +1,7 @@
+# Why these reducers were extracted from the root reducer
```

Comment:
Why not submit a GitHub issue instead of adding documentation for this?
If you feel you need to document it, a few well-placed inline comments would do.

Reply:
https://github.com/Automattic/wp-calypso/issues/18180